pkg/apis/v1/nodepool_conversion_test.go

```diff
@@ -269,6 +270,24 @@ var _ = Describe("Convert V1 to V1beta1 NodePool API", func() {
 			Expect(v1beta1nodepool.Status.Resources[resource]).To(Equal(v1nodepool.Status.Resources[resource]))
 		}
 	})
+	Context("Round Trip", func() {
+		DescribeTable(
+			"NillableDuration",
+			func(value string, field func() *NillableDuration) {
+				str := fmt.Sprintf("%q", value)
+				duration := NillableDuration{}
+				Expect(json.Unmarshal([]byte(str), &duration)).Should(Succeed())
+				*field() = duration
```

Comment:
I don't quite understand what this is doing, is this setting the output of the return value of the func to duration?

Reply:
Removed this indirection, more confusing than it was worth. This was just a way to mutate an arbitrary field in the NodePool, specified by the Entry in the table.